### ISSUE TITLE
Crash on application start Failed resolution of: Landroid/support/v4/util/ArraySet

### ISSUE THREAD

**Issue by thunderboot** [playgameservices/play-games-plugin-for-unity#2617]
I am trying to use both Google Play Services and the AdMob plugin in Unity 2019.1.8f1, building an APK works, but running it on a device results in a crash.

I have attached a log from LogCat and I think this is what causes the crash:
`java.lang.NoClassDefFoundError: Failed resolution of: Landroid/support/v4/util/ArraySet;`

I am using [CloudOnce](https://github.com/jizc/CloudOnce) v2.6.8 which in turn uses the play-games-plugin-for-unity v0.9.64 and I am trying to add the admob plugin v.3.18.

I have also uploaded a [test project](https://drive.google.com/file/d/18fhFW_ltnqM3wgJC5F0sVhVC7A5NdBN4/view?usp=sharing) which reproduces the error.

I have tried troubleshooting, but not really gotten anywhere, only found that there have been some commits to the play-games-plugin-for-unity since v0.9.64 was released. Since one change is replacing 
`import android.support.annotation.NonNull;`
with
`import androidx.annotation.NonNull;`
is that something that might fix this problem since the error in the log references android/support/v4/....?

I have not been able to test my theory since I don't know how to build a unitypackage from the new source files above. I can test if someone instructs me how to do it or is a new release on its way already? :)

[CloudOnce_AdMob_testlog.txt](https://github.com/playgameservices/play-games-plugin-for-unity/files/3337471/CloudOnce_AdMob_testlog.txt)

**Comment by acsudeep** [comment]
I am having same error as well. Running Force resolve does not works at all.

**Comment by Norbinarius** [comment]
Solved this issue by using admob plugin v3.17 instead of v3.18

**Comment by acsudeep** [comment]
What procedure did you used, i mean, did you uploaded the playservice service first of admob?

**Comment by thunderboot** [comment]
@Norbinarius Same here. I imported the latest version of CloudOnce, downloaded the latest version of PlayServicesResolver and imported it (overwriting the one from CloudOnce) and finally imported AdMob 3.17 (NOT importing the PlayServicesResolver). 

I then checked [Wayback Machine](https://web.archive.org/web/20190508181942/https://developers.google.com/admob/unity/start) from May the 8th 2019 to find the instructions for how to configure the app ids in AdMob with the Unity Editor plugin.

This will work for now, but play-games-plugin-for-unity still needs to be updated to androidx.
